library/rating/rating.css

```diff
@@ -0,0 +1,30 @@
+.stars {
+  height: 10rem;
+  text-align: center;
+}
+
+.stars input {
+  display: none;
+}
+
+.stars label {
+  float: right;
+  font-size: 4rem;
+  color: var(--white-color);
+  margin: 0 0.5rem;
```

Comment:
use pixels in place of 0.5rem.

Reply:
Have been using the same throughout the project, reason being i use 16px = 1rem as base. So here my calculation is easy with 0.5rem = 8px.